.travis.yml

```diff
@@ -1,20 +1,35 @@
-services: docker
+dist: "trusty"
+sudo: required
+
+matrix:
+  include:
+    - language: python
+      python: 3.5
 
 language: ruby
+addons:
+  apt:
+    packages:
+      - "python3-pip"
+
 rvm:
   - 2.3.3
 cache: bundler
 
+before_script:
+  - bundle install
+  - sudo -H pip3 install --upgrade pip
```

Comment:
no sudo pls.

Anyways ... after https://github.com/coala/projects/pull/593 it is easy to add another Python job.

Reply:
Here pip install didn't work without sudo. Perhaps I will make these changes after #593 is merged.

Reply:
It was merged